src/components/StatusIndicator/index.tsx

```diff
@@ -1,11 +1,25 @@
 import { css } from '@emotion/react'
+import styled from '@emotion/styled'
 import PropTypes from 'prop-types'
-import { ColorDeprecated as Color } from '../../theme/deprecated/colors'
+import { Color } from '../../theme'
+import { ColorDeprecated } from '../../theme/deprecated/colors'
 import { flash } from '../../utils/animations'
-import { XStyledProps } from '../Box'
-import Dot from '../Dot'
+import Box, { XStyledProps } from '../Box'
 import Tooltip from '../Tooltip'
 
+const Dot = styled(Box, {
```

Comment:
Styled div ?

Reply:
No we are passing multiple props in our console like margin and padding, because of using it with a prop spreading i think it can have lot of impact on our console @chambo-e :/